code/img2text/main.py

```diff
@@ -194,6 +195,29 @@ def extract_nouns(text):
         max_key = max(category_similarities, key=category_similarities.get)
         return max_key
 
-output = PredictCategory()
-print(output.get_predict())
+# output = PredictCategory()
+# print(output.get_predict())
 
+class BooksRecommendation():
+    def __init__(self):
+        base_url = "https://app.rakuten.co.jp/services/api/BooksTotal/Search/20170404?applicationId=1028959429215953336"
+        output = PredictCategory()
+        new_keyword = output.get_predict()
+        new_keyword = "Health"
+        url = f"{base_url}&keyword={new_keyword}&sort=%2BitemPrice"
+
+        response = requests.get(url)
+        self.json_response = response.json()
+
+    def get_recommend(self):
+        recommendBooks = {"recommendBooks": []}
+        for i in range(30):
+            title = self.json_response['Items'][i]['Item']['title'].replace('\u3000', '')
+            isbn = self.json_response['Items'][i]['Item']['isbn']
+            item_url = self.json_response['Items'][i]['Item']['itemUrl']
+            item_price = self.json_response['Items'][i]['Item']['itemPrice']
+            recommendBooks["recommendBooks"].append({"title": title, "isbn": isbn, "itemUrl": item_url, "item_price": item_price})
```

Comment:
```suggestion
            item = self.json_response['Items'][i]['Item']
            title = item['title'].replace('\u3000', '')
            isbn = item['isbn']
            item_url = item['itemUrl']
            item_price = item['itemPrice']
```

一旦itemっていう説明変数に格納すると可読性向上するかも？？
余計なお世話すみません🙏これは無視してマージしてもらって結構です🙏
てか倫明マジですごい！！ここまでカバーしてくれるの超助かります🙇‍♂️